src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/ViewModels/DataSetVersionChangesViewModel.cs

```diff
@@ -5,10 +5,17 @@ namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Api.ViewModels;
 /// <summary>
 /// A set of changes made to a data set version.
 /// </summary>
-public record DataSetVersionChangesViewModel
+public record DataSetVersionChangeSets
```

Comment:
I'd be tempted to rename this to `DataSetVersionChangeSet`, dropping the plural?